eskipfile/remote.go

```diff
@@ -172,13 +182,31 @@ func (client *remoteEskipFile) DownloadRemoteFile() error {
 }
 
 func (client *remoteEskipFile) getRemoteData() (io.ReadCloser, error) {
-	resp, err := client.http.Get(client.remotePath)
+	req, err := http.NewRequest("GET", client.remotePath, nil)
+
 	if err != nil {
 		return nil, err
 	}
+
+	if len(client.etag) != 0 {
+		req.Header.Set("If-None-Match", client.etag)
+	}
+
+	resp, err := client.http.Do(req)
+	if err != nil {
+		return nil, err
+	}
+
+	if resp.StatusCode == 304 {
+		client.onCachedSometimes.Do(func() { log.Info("remote routes are cached") })
```

Comment:
Why do we want to log this?

Reply:
I thought we may need to log every now and then that we use cached version.

Reply:
Would you prefer if I remove it?